internal/command/command_test.go

```diff
@@ -618,14 +620,21 @@ func testStdoutCapture(t *testing.T, dst io.Writer) func() {
 	doneCh := make(chan struct{})
 	go func() {
 		defer close(doneCh)
-		defer r.Close()
-		io.Copy(dst, r)
+		if _, err := io.Copy(dst, r); err != nil {
+			panic(err)
+		}
+		if err := r.Close(); err != nil {
+			panic(err)
+		}
 	}()
 
 	return func() {
 		// Close the writer end of the pipe
-		w.Sync()
-		w.Close()
+		// This test code is racey
+		_ = w.Sync()
+		if err := w.Close(); err != nil {
+			t.Fatal(err)
```

Comment:
Could we replace these with specific messages for an easier identification of the failing cause?
```suggestion
			t.Fatalf("closing the pipe writer failed: %w", err)
```

Reply:
Fatal will panic and show the line-number anyways.  I prefer keeping the tests as simple as possible.

Reply:
Oh, I forgot that it shows the line number. Good point. Thanks.